lib/svrquery/protocol/titanfall/types.go

```diff
@@ -33,11 +38,21 @@ func (i Info) NumClients() int64 {
 	return int64(i.BasicInfo.NumClients)
 }
 
+// NumBotClients implements protocol.Responser.
+func (i Info) NumBotClients() int64 {
+	return int64(i.BasicInfo.NumBotClients)
+}
+
 // MaxClients implements protocol.Responser.
 func (i Info) MaxClients() int64 {
 	return int64(i.BasicInfo.MaxClients)
 }
 
+// TotalClientsConnectedEver implements protocol.Responser.
```

Comment:
As above, is it a problem that this is zero for older versions vs not implementing the interface?